extra-dev/packages/coq-goedel/coq-goedel.dev/opam

```diff
@@ -12,10 +12,11 @@ A proof in Coq of the Gödel-Rosser 1st incompleteness theorem,
 which says that any first order theory extending NN (which is PA
 without induction) that is complete is inconsistent."""
 
-build: [make "-j%{jobs}%"]
-install: [make "install"]
+build: ["dune" "build" "-p" name "-j" jobs]
 depends: [
+  "dune" {>= "2.5"}
   "coq" {(>= "8.7" & < "8.14~") | (= "dev")}
+  "coq-ackermann" {= version}
```

Comment:
Any reason to use `version` rather than `"dev"` here? (Trying to understand why `coq-pocklington` would be `= "dev"`, while `coq-ackermann` would be `= version`.)

Reply:
The main reason is that `coq-goedel` and `coq-ackermann` live in the same Git repository (and the same archive), so it makes sense to tie them together fully. On the other hand, it may happen that `coq-goedel` is compatible with a range of versions of `coq-pocklington`, not just the  `dev` version (even if it only works with `dev` now).